mlflow/server/js/src/common/components/error-boundaries/AppErrorBoundary.js

```diff
@@ -24,17 +24,25 @@ class AppErrorBoundary extends Component {
   }
 
   componentDidCatch(error, errorInfo) {
-    this.setState({ hasError: true });
+    this.setState({ hasError: true, error });
     console.error(error, errorInfo);
   }
 
   render() {
+    const { error } = this.state;
     return (
       <>
         {this.state.hasError ? (
           <div>
             <img className='error-image' alt='Error' src={defaultErrorImg} />
-            <h1 className={'center'}>Something went wrong</h1>
+            <h1 className={'center'}>Something went wrong: {error.toString()}</h1>
+            <Accordion>
+              <Accordion.Panel header='Show details'>
+                <Typography css={{ fontSize: 10, whiteSpace: 'pre-wrap' }}>
+                  {error.stack.toString()}
+                </Typography>
+              </Accordion.Panel>
+            </Accordion>
```

Comment:
Is it possible to un-minify the stack trace?

<img width="1096" alt="image" src="https://user-images.githubusercontent.com/17039389/223110031-e0183411-9f3c-4d87-8d88-9ad886a3f760.png">

Reply:
Maybe just adding `Open the browser console to see what went wrong` is enough.

Reply:
🤔 just my thoughts: actually with the stack trace below, a user won't need to open the browser console because everything should be in UI anyway. Actually I'd leave just "Something went wrong" with "Show details" below and that's it.

Reply:
Is this something that should just be for OSS rather than databricks? Unless thats already the case where `mlflow/server/js/src/common/components/error-boundaries/AppErrorBoundary.js` is only part of OSS? I don't remember that.

Also why I am suggesting this to be just OSS since that is useful for OSS to create issues. But in Databricks we already create issues if there are such errors. Just a thought.